x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/test/WatcherTestUtils.java

```diff
@@ -149,24 +152,25 @@ public static WatchExecutionContext createWatchExecutionContext() throws Excepti
 
 
     public static Watch createTestWatch(String watchName, Client client, HttpClient httpClient, EmailService emailService,
-                                        WatcherSearchTemplateService searchTemplateService, Logger logger) throws AddressException {
+                                        WatcherSearchTemplateService searchTemplateService, Logger logger) {
+        ActionThrottler actionThrottler = new ActionThrottler(Clock.systemUTC(), null, mock(XPackLicenseState.class));
```

Comment:
What is this change for ?

Reply:
So the reason this change is needed, is because the `ListWatchesResponseTests` creates watch instances for testing xcontent serialization and a couple more components are needed to make a watch xcontent serialization work without NPE's being thrown.